src/views/Card.test.jsx

```diff
@@ -0,0 +1,1090 @@
+        appeardAnime: '43',
+        clowCard: 'https://i.ibb.co/NVc0wX3/Gemelos.jpg',
+        sakuraCard: 'https://i.ibb.co/MBSwTL4/Gemelos-Sakura.jpg',
+        meaning: ' Simbolizan la dualidad, la compañía, la multiplicación.',
+        __v: 0,
+        cardsReverse: {
+          clowReverse: 'https://i.ibb.co/LJSmQ4f/Reverso-Clow.jpg',
+          sakuraReverse: 'https://i.ibb.co/XxrvMJ2/Reverso-Sakura.jpg',
+        },
+      },
+      {
+        _id: '603a8c063da2c60015bd6942',
+        cardNumber: 54,
+        spanishName: 'La Nada',
+        englishName: 'The Nothing',
+        kanji: '無',
+        Rōmaji: 'Mu',
+        appeardManga: '',
+        appeardAnime: 'Movie n° 2',
+        clowCard: 'https://i.ibb.co/gPXkCJZ/Nada.jpg',
+        sakuraCard: 'https://i.ibb.co/zG7fpxp/Nada-Sakura.jpg',
+        meaning: 'Simboliza el espacio vacío, la nada, lo negativo.',
+        __v: 0,
+        cardsReverse: {
+          clowReverse: 'https://i.ibb.co/LJSmQ4f/Reverso-Clow.jpg',
+          sakuraReverse: 'https://i.ibb.co/XxrvMJ2/Reverso-Sakura.jpg',
+        },
+      },
+      {
+        _id: '603a8c993da2c60015bd6943',
+        cardNumber: 55,
+        spanishName: 'La Esperanza',
+        englishName: 'The Hope',
+        kanji: '希望',
+        Rōmaji: 'Kibo',
+        appeardManga: '',
+        appeardAnime: 'Movie n° 2',
+        clowCard: '',
+        sakuraCard: 'https://i.ibb.co/sPmXJ7n/La-esperanza.jpg',
+        meaning: 'Simboliza la esperanza de un amor.',
+        __v: 0,
+        cardsReverse: {
+          sakuraReverse: 'https://i.ibb.co/XxrvMJ2/Reverso-Sakura.jpg',
+        },
+      },
+      {
+        _id: '618a0cc5c0e81c0016b8c3a3',
+        __v: 0,
+      },
+      {
+        _id: '618a0cd2c0e81c0016b8c3a4',
+        __v: 0,
+      },
+      {
+        _id: '618a0cd4c0e81c0016b8c3a5',
+        __v: 0,
+      },
+      {
+        _id: '618a0cd5c0e81c0016b8c3a6',
+        __v: 0,
+      },
+      {
+        _id: '618a0cd7c0e81c0016b8c3a7',
+        __v: 0,
+      },
+    ],
+    page: 1,
+    pageSize: 60,
+    count: 60,
+    totalCount: 65,
+  },
+];
+
+const server = setupServer(
+  rest.get(
+    'https://protected-taiga-89091.herokuapp.com/api/card',
+    (req, res, ctx) => {
+      return res(ctx.json(cards));
+    }
+  )
+);
+
+beforeAll(() => server.listen());
+afterAll(() => server.close());
+
+test('renders detail when routing', async () => {
+  render(
+    <MemoryRouter initialEntries={['/6039396a68347a4a842920cf']}>
+      <Route path="/:_id">
+        <Card />
+      </Route>
+    </MemoryRouter>
+  );
+  const item = await screen.findByText('Spanish Name: Viento');
+  expect(item).toBeInTheDocument();
+});
+
+test('goes back to home loading', async () => {
+  render(
+    <MemoryRouter initialEntries={['/6039396a68347a4a842920cf']}>
+      <Route path="/:_id">
+        <Card />
+      </Route>
+    </MemoryRouter>
+  );
+  const back = await screen.findByText('back');
+  userEvent.click(back);
+  render(
```

Comment:
I'm not sure that I would test in this fashion, it may make more sense to render a common parent to both `Card` and `Cards` to test